content/asa/0003-tecnicas-algoritmos.md

```diff
@@ -436,8 +436,7 @@ O código para este algoritmo seria qualquer coisa como:
 int fracKnapsack(std::vector<int> values,
                  std::vector<int> weights,
                  int maxWeight) {
-  // Assumimos, aqui, que os objetos estão ordenados
-  // por ordem decrescente de densidade
+  // Assumimos, aqui, que os objetos estão ordenados por ordem decrescente de densidade
```

Comment:
Change this back, now it doesn't fit the width of the page and shows horizontal scroll on the code block

Reply:
Bah, must've accepted the wrong one while fixing merge conflicts, my bad